rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -411,6 +411,7 @@ maas_apt_packages:
 maas_pip_packages:
   - rackspace-monitoring-cli
   - ipaddr
+  - swift
```

Comment:
in kilo, swift got installed on the host (as a pip package) by the os_swift role from osa.  I think this was probably a side effect more than anything on purpose. Now that things have moved into venvs, this no longer gets done, so we need to explicitly install it for maas.

See https://gist.github.com/mancdaz/6052ed1810ca1c521b41